Makefile

```diff
@@ -166,6 +167,10 @@ cephcsi: check-env
 e2e.test: check-env
 	go test $(GO_TAGS) -mod=vendor -c ./e2e
 
+.PHONY: rbd-group-snapshot
+rbd-group-snapshot:
```

Comment:
@nixpanic i hope this is not required

Reply:
no, that is part of testing I'm doing. But it seems to be included in the devel branch too?

Reply:
i think it got in as part of this PR https://github.com/ceph/ceph-csi/pull/4473/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R169 which we should not have done